packages/kbn-securitysolution-utils/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@kbn/babel-preset/node_preset"]
```

Comment:
```suggestion
  "presets": ["@kbn/babel-preset/node_preset"],
  "ignore": ["**/*.test.ts", "**/*.test.tsx"]
```

Did you want to the line `"ignore": ["**/*.test.ts", "**/*.test.tsx"]` like the others?

Reply:
👍